src/ClientModel.cpp

```diff
@@ -274,6 +274,12 @@ void ClientModel::setupScenario(QVariantMap _scenario)
 
 	m_ethAccounts->setAccounts(m_accountsSecret);
 
+	//m_ethAccounts = make_shared<FixedAccountHolder>([=](){return m_client.get();}, std::vector<KeyPair>());
+	m_web3Server.reset(new Web3Server(*m_rpcConnector.get(), m_ethAccounts, std::vector<KeyPair>(), m_client.get()));
```

Comment:
cause m_ethAccounts contains list of user added by the user to the "genesis block". 
We need to be able to use those accounts in web3.